api/types.go

```diff
@@ -171,11 +171,6 @@ type Runner struct {
 	UseMMap   bool `json:"use_mmap,omitempty"`
 	UseMLock  bool `json:"use_mlock,omitempty"`
 	NumThread int  `json:"num_thread,omitempty"`
-
-	// Unused: RopeFrequencyBase is ignored. Instead the value in the model will be used
-	RopeFrequencyBase float32 `json:"rope_frequency_base,omitempty"`
```

Comment:
Worth calling out this was left in the struct because some older models still set this parameter and removing it caused those models to not load. The same is likely true for NumGQA